base/client/src/main/java/dev/rico/client/projector/Projector.java

```diff
@@ -33,9 +34,7 @@
 
     ControllerProxy<? extends ManagedUiModel> getControllerProxy();
 
-    @Deprecated
     Map<IdentifiableModel, Node> getModelToNodeMap();
 
-    @Deprecated
```

Comment:
see above

Reply:
I am fine with making this deprecated. Nevertheless we need a way that enables us to customize the generated javafx nodes.

Reply:
referenced for discussion in #27